site/cypress/e2e/Cypress-UI/profile.spec.js

```diff
@@ -85,11 +85,52 @@ describe('Test cases revolving around user profile page', () => {
 
     // Selenium test_time_zone_selection
     it('Time zone selector should work', () => {
-        cy.get('[data-testid="time-zone-dropdown"]').should('contain.text', 'NOT_SET/NOT_SET');
-        cy.get('[data-testid="time-zone-dropdown"]').select('(UTC-07:00) America/Phoenix');
-        // Since the login success message is still up, we get the next message.
-        cy.get('[data-testid="popup-message"]').next().should('contain.text', 'Time-zone updated successfully');
-        cy.get('[data-testid="time-zone-dropdown"]').select('NOT_SET/NOT_SET');
+
+        //Check that the default value is NOT_SET/NOT_SET
+        it('should have the correct default value', () => {
+            cy.get('[data-testid="time-zone-dropdown"]').should('contain.text', 'NOT_SET/NOT_SET');
```

Comment:
One line tests in cypress are inefficient, and not good practice. In reality, all of these separate 'it' statements can be combined into one.